utilities/test_suite/HOST/runImageTests.py

```diff
@@ -40,7 +40,7 @@
 outFolderPath = os.getcwd()
 buildFolderPath = os.getcwd()
 caseMin = 0
-caseMax = 92
+caseMax = 93
```

Comment:
I have commented this before. We need to use enums for testcases instead of hardcoded values